src/answers-umd.js

```diff
@@ -433,7 +432,7 @@ class Answers {
     try {
       this.components.create(type, opts).mount();
     } catch (e) {
-      throw new AnswersComponentError('Failed to add component', type, e);
+      console.error('Failed to add component', type, e);
```

Comment:
nit: I'm not sure what the output looks like, but maybe we should put the error message on a separate line?

Reply:
We added two new lines so its prettier now.